docs/appendix/zowe-glossary.md

```diff
@@ -0,0 +1,188 @@
+
+* **Zowe sample library**
+
+  (See also SZWESAMP) – Does it need to go in Glossary?
+
+* **ZWESVUSR** 
+
+  This is a started task ID used to run the PROCLIB ZWESVSTC. The task starts a USS environment using BPXBATSL that executes the core Zowe Desktop (ZLUX) node.js server, the Java API Mediation Layer, and the Z Secure Services C component. To work with USS, the user ID ZWESVUSR must have a valid OMVS segment. For more information, see System requirements | Zowe Docs
+
+* **Zowe runtime directory**
+
+  referred to as `<RUNTIME_DIR>`. Also known as ROOT DIR is the directory where the Zowe runtime is located, also referred to as the <RUNTIME_DIR>. Defaults to the location of where zowe-configure-instance was executed. See Creating and configuring the Zowe instance directory | Zowe Docs 
+
+## Operation
+
+* **App2App**
+
+  It is a unique feature to Zowe environment where one application plug-in can communicate with another and it is applicable to multiple application plug-ins. The application framework provides constructs that facilitate this ability. See Application-to-application communication | Zowe DocsConfig Service: 
+
+* **MVS (multiple virtual storage) Explorer** 
+
+  view z/OS data sets from the Zowe Desktop
+
+* **USS Explorer** 
+
+  a plug-in provided with zowe to view z/OS UNIX files from the Zowe Desktop
+
+* **File Explorer**
+
+  a plug-in provided with Zowe to allow users to view z/OS files and data sets using a RESTful interface. Also used by the MVS Explorer UI and USS Explorer UI
+
+* **JES Explorer** 
+
+
+## Tools & Plug-ins
+
+* **Configuration file**
+ 
+* **API ML Plain Java Enabler (PJE)**
+
+  is a library which helps to simplify the process of onboarding a REST service with the API ML
+
+* **Service**
+
+  A service provides one or more APIs, and is identified by a service ID. Note that sometimes the term "service name" is used to mean service ID.The default service ID is provided by the service developer in the service configuration file.A system administrator can replace the service ID with a deployment environment specific name using additional configuration that is external to the service deployment unit. Most often, this is configured in a JAR or WAR file.Services are deployed using one or more service instances, which share the same service ID and implementation.
+
+* **Extension** 
+
+* **Plug-in** 
+
+
+## Community
+
+* **OMP/Open Mainframe Project**
+  
+  The organization the Zowe project is a member of
+
+* **LF/Linux Foundation**
+
+  The organization the OMP is a member of it.
```

Comment:
Wikipedia says:
The Linux Foundation (LF) is a technology consortium... to standardize Linux, support its growth, and promote its commercial adoption. Additionally, it hosts and promotes the collaborative development of open source software projects.
...and yes, the OMP is one of those open source software projects.